src/main/java/se/sundsvall/messaging/api/model/response/UserMessages.java

```diff
@@ -0,0 +1,27 @@
+package se.sundsvall.messaging.api.model.response;
+
+import static io.swagger.v3.oas.annotations.media.Schema.AccessMode.READ_ONLY;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import se.sundsvall.dept44.models.api.paging.PagingMetaData;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+
+@Builder(setterPrefix = "with")
+@Schema(description = "User messages model")
+public record UserMessages(
+
+	@JsonProperty("_meta")
+	@Schema(implementation = PagingMetaData.class, accessMode = READ_ONLY)
+	PagingMetaData metaData,
```

Comment:
Don´t you need some paging possibility (by request parameters for example) in the request if the response is paged (i,e, if client wants to fetch next page and so on)?

Reply:
If you look closely, there is pagination in the request.